content/docs/command-reference/cache/dir.md

```diff
@@ -24,22 +24,31 @@ Unlike doing so with `dvc config cache`, `dvc cache dir` transform paths
 **relative to the config file location**. However, if the `value` provided is an
 absolute path, then it's preserved as it is.
 
-If no path `value` is provided to this command, it prints the path for current
-cache directory.
+When reading (`value` is not provided) the value is read from the system,
+global, repo and local configuration files by default, and options `--system`,
+`--global`, `--repo`, and `--local` can be used to tell the command to read from
+only that location.
```

Comment:
```suggestion
If no path `value` is provided to this command, it prints the path for current
cache directory. the value is searched in the system, global, repo, and local
config files by default (in that order). Options `--system`, `--global`,
`--repo`, and `--local` can be used to look only in one place.
```

Reply:
> Options `--system`, `--global`, `--repo`, and `--local` can be used to look only in one place.

If we want to simplify the description, we can skip this since it will be explained in the **Options** section.

Reply:
> since it will be explained

Actually, I see it's not exactly specified there, but it could (probably should) — again, to avoid overcomplicating the description.

Reply:
The original text is adapted from git docs, but ok, let's have it this way.

Reply:
Thanks. I don't think git docs are usually super relevant but... that can definitely be a great starting point 👍